cli/src/datafeed.ts

```diff
@@ -195,6 +224,168 @@ export function bulkRemoveHandler() {
+    const bulkData = data as IDatafeedBulk
+
+    const chain = bulkData?.chain || 'localhost'
+    const service = bulkData?.service || 'DATA_FEED'
+
+    const fetcherHost = bulkData?.fetcherHost || 'http://fetcher.orakl.svc.cluster.local'
+    const workerHost = bulkData?.workerHost || 'http://aggregator-worker.orakl.svc.cluster.local'
+    const listenerHost =
+      bulkData?.listenerHost || 'http://aggregator-listener.orakl.svc.cluster.local'
+    const reporterHost =
+      bulkData?.reporterHost || 'http://aggregator-reporter.orakl.svc.cluster.local'
+
+    const fetcherPort = bulkData?.fetcherPort || '4040'
+    const workerPort = bulkData?.workerPort || '5000'
+    const listenerPort = bulkData?.listenerPort || '4000'
+    const reporterPort = bulkData?.reporterPort || '6000'
+
+    const listeners = await listenerListHandler()({ chain, service })
+    const reporters = await reporterListHandler()({ chain, service })
+
+    if (!checkBulkSource(data?.bulk)) {
+      console.error(`invalid json src format`)
+      return
+    }
+
+    for (const activateElement of data.bulk) {
+      const aggregatorData = await loadJsonFromUrl(activateElement.aggregatorSource)
+      if (!checkAggregatorSource(aggregatorData)) {
+        console.error(`invalid aggregatorData: ${aggregatorData}, skipping activation`)
+        continue
+      }
+
+      const reporterId = reporters.find(
+        (reporter) =>
+          reporter.address.toLowerCase() == activateElement.reporter.walletAddress.toLowerCase()
+      )?.id
+      if (!reporterId) {
+        console.error(
+          `reporterId not found for ${activateElement.reporter.walletAddress}, skipping activation`
+        )
+        continue
+      }
+
+      const listenerId = listeners.find(
+        (listener) => listener.address == aggregatorData.address
+      )?.id
+      if (!listenerId) {
+        console.error(`listenerId not found for ${aggregatorData.address}, skipping activation`)
+        continue
+      }
+
+      await fetcherStartHandler()({
+        id: aggregatorData.aggregatorHash,
+        chain,
+        host: fetcherHost,
+        port: fetcherPort
+      })
+      await aggregatorActivateHandler()({
+        aggregatorHash: aggregatorData.aggregatorHash,
+        host: workerHost,
+        port: workerPort
+      })
+
+      await reporterActivateHandler()({
+        id: Number(reporterId),
+        host: reporterHost,
+        port: reporterPort
+      })
+      await listenerActivateHandler()({
+        id: Number(listenerId),
+        host: listenerHost,
+        port: listenerPort
+      })
+    }
+  }
+  return wrapper
+}
+
+export function bulkDeactivateHandler() {
+  async function wrapper({ data }: { data }) {
+    const bulkData = data as IDatafeedBulk
+
+    const chain = bulkData?.chain || 'localhost'
+    const service = bulkData?.service || 'DATA_FEED'
+
+    const fetcherHost = bulkData?.fetcherHost || 'http://fetcher.orakl.svc.cluster.local'
```

Comment:
@nick-bisonai How about loading the default values from https://github.com/Bisonai/orakl/blob/master/cli/src/settings.ts? We could then easily update the values for both activation, deactivation. It could be also good to apply for ports.

Reply:
unfortunately, I don't think it matches actual settings in our k8s and it seem to be set for local environment.
should I change settings.ts?

Reply:
if .env file is set for the pod I think it'd okay I'll reference settings.ts as you've recommended

Reply:
We could have two versions: local and production with default values for both. It looks ugly, but would do the job 🤔 

```typescript
const production  = process.env.NODE_ENV == "production" ? true : false
const defaultWorkerServicePort = process.env.WORKER_SERVICE_PORT
export const WORKER_SERVICE_PORT = defaultWorkerServicePort || production ? 5000 : 5001
```